Zend/zend_builtin_functions.c

```diff
@@ -1715,6 +1715,18 @@ ZEND_FUNCTION(get_defined_constants)
 }
 /* }}} */
 
+static bool backtrace_is_arg_sensitive(zend_execute_data *call, uint32_t offset)
```

Comment:
With #20448 the following should be possible:
```suggestion
static bool backtrace_is_arg_sensitive(const zend_execute_data *call, uint32_t offset)
```